libopenage/renderer/opengl/texture.cpp

```diff
@@ -106,11 +114,9 @@ void GlTexture2d::upload(resources::Texture2dData const& data) {
 	auto size = this->info.get_size();
 	auto fmt_in_out = GL_PIXEL_FORMAT.get(this->info.get_format());
 
-	glTexSubImage2D(GL_TEXTURE_2D, 0, 0, 0,
-	                size.first, size.second,
-	                std::get<1>(fmt_in_out), std::get<2>(fmt_in_out),
-	                data.get_data()
-	);
+	glTexSubImage2D(GL_TEXTURE_2D, 0, 0, 0, size.first, size.second, std::get<1>(fmt_in_out), std::get<2>(fmt_in_out), data.get_data());
 }
 
-}}} // openage::renderer::opengl
+} // namespace opengl
+} // namespace renderer
+} // namespace openage
```

Comment:
actually, there is a change because I changed the filter function to `GL_NEAREST`

https://github.com/SFTtech/openage/pull/1388/files/97c60b1ba209d20e270825f6995fef28ba6c159d#diff-b3f9fcb2ec8c9f8cf2cab00a72ea3d7690f429c7de12dc7f863d14d465f62542L51-R55